Phonebook.Frontend/src/app/pages/dashboard/components/team/team.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Select, Store } from '@ngxs/store';
+import { Observable, Subscription } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Person, PhonebookSortDirection } from 'src/app/shared/models';
+import { BookmarksState } from 'src/app/shared/states';
+import { OrganigramService, UnitTreeNode } from 'src/app/services/api/organigram.service';
+import { CurrentUserService } from 'src/app/services/api/current-user.service';
+import { untilComponentDestroyed } from 'ng2-rx-componentdestroyed';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-team',
+  templateUrl: './team.component.html',
+  styleUrls: ['./team.component.scss'],
+  host: { class: 'pb-expand' }
+})
+export class TeamComponent implements OnInit, OnDestroy {
+  public bookmarkedPersons: Person[];
```

Comment:
remove properties and funtion not needed. (e.g. from bookmarks component)

Reply:
These properties and the function are actually needed. If I remove them, the text explaining how to bookmark People won't show up next to bookmarks when there are no bookmarks.

Reply:
Then remove it after moving the navigation to the dashboard